moveit_plugins/moveit_simple_controller_manager/include/moveit_simple_controller_manager/action_based_controller_handle.h

```diff
@@ -206,21 +204,40 @@ class ActionBasedControllerHandle : public ActionBasedControllerHandleBase
     done_ = true;
   }
 
-  /* execution status */
+  /**
+   * @brief Node used to create the action client.
+   */
+  const rclcpp::Node::SharedPtr node_;
```

Comment:
It looks like `node_` is hardly used at all. Delete it

Reply:
Deleted!